spec/models/admin_ability_spec.rb

```diff
@@ -500,6 +508,7 @@
       it{ should be_able_to(:show, my_conference.program) }
       it{ should be_able_to(:update, new_track) }
       it{ should be_able_to(:manage, my_self_organized_track) }
+      it{ should_not be_able_to(:admins, organization) }
```

Comment:
these test should also check that non-admins cannot `unassign_org_admins` or `assign_org_admins`, no?